src/Middleware/RecordForegroundTransaction.php

```diff
@@ -84,7 +85,7 @@ public function terminate($request, $response)
             new RequestProcessedSpan(
                 $this->getTransactionName($request),
                 [
-                    'now' => now()->toDateTimeString(),
+                    'now' => Carbon::now(),
```

Comment:
Make it `Carbon::now()->toDateTimeString()` please.

Reply:
Done